spec/models/comment_spec.rb

```diff
@@ -85,7 +85,7 @@
       end
     end
 
-    describe "#mention_total" do
+    describe "#user_mentions_in_markdown" do
```

Comment:
This just updates the description in the `comment_spec`. I referenced it when writing specs for this and noticed that the `describe` block was referencing a method that didn't even exist 😅